glusterd2/commands/volumes/volume-status.go

```diff
@@ -12,6 +12,41 @@ import (
 	"github.com/gorilla/mux"
 )
 
+func volumeStatusAllHandler(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	logger := gdctx.GetReqLogger(ctx)
+
+	volumes, e := volume.GetVolumes()
+	if e != nil {
+		logger.WithError(e).Error("Failed to get volumes")
+		restutils.SendHTTPError(ctx, w, http.StatusBadRequest, errors.ErrVolNotStarted)
```

Comment:
IMO it is a InternalError and the error is not VolumeNotStarted. Something like https://github.com/gluster/glusterd2/blob/master/glusterd2/commands/peers/getpeers.go#L26